### ISSUE TITLE
loadConfig: either bitcoin.active or litecoin.active must be set to 1 (true)

### ISSUE THREAD

**Issue by OvoNemaSmisla** [lncm/docker-lnd#42]
Hi,

After pulling the latest version `docker pull lncm/lnd:v0.10.0`, I've tried to run it with:
```
docker run  -it  --rm  --detach \
    -v ~/.lnd:/data/.lnd \
    -p 9735:9735 \
    -p 10009:10009 \
    --name lnd \
    lncm/lnd:v0.10.0
```
I get the following error:
`loadConfig: either bitcoin.active or litecoin.active must be set to 1 (true)`

This is happening on Ubuntu 18.04.1, 5.3.0-53-generic, x86_64 arch and Ubuntu 18.04.3,  4.9.219-72, aarch64.

**Comment by nolim1t** [comment]
Hi,

Can you share a redacted version of your LND.conf

in ```~/.lnd/data``` you need to have a lnd.conf file

**Comment by OvoNemaSmisla** [comment]
`~/.lnd` direcotry ownership is root:root with 0755 permissions and is empty on both systems.

**Comment by nolim1t** [comment]
you need to have a valid lnd.conf file or specify the appropriate command switches

**Comment by OvoNemaSmisla** [comment]
I assume the config file is something like bitcoin.conf for the BTC node?

If so, is there something similar to [this](https://github.com/bitcoin/bitcoin/blob/0212187fc624ea4a02fc99bc57ebd413499a9ee1/contrib/debian/examples/bitcoin.conf) for me to understand what are the options/parameters?

**Comment by nolim1t** [comment]
no its your LND configuration.

**Comment by nolim1t** [comment]
Is this still an issue?

**Comment by nolim1t** [comment]
please refer to [this guide](https://github.com/lncm/docker-lnd/blob/38a9f31129a97139fd1628d0d92aae708d721179/README.md#start) for how to configure.

**Comment by OvoNemaSmisla** [comment]
Thanks for pointing me in the right direction, I'll try setting up a proper config file and report the results.

**Comment by meeDamian** [comment]
Setting up Docker volume correctly is the only thing within the scope of this repo, everything else relates to upstream's config.

I'm going to close this one, as @nolim1t has addressed it well already, #48 exists, and any further config discussion belongs to [lnd's](https://github.com/lightningnetwork/lnd) main repo.
